package.json

```diff
@@ -25,20 +25,21 @@
     "@babel/core": "7.6.0",
     "@babel/preset-env": "7.2.3",
     "@babel/preset-react": "7.0.0",
-    "@ledgerhq/hw-transport": "4.48.0",
-    "@ledgerhq/hw-transport-u2f": "4.48.0",
+    "@ledgerhq/hw-transport": "5.0.0-babel7.3",
```

Comment:
These need to be updated due to this issue that's been open for a long time: https://github.com/LedgerHQ/ledgerjs/issues/332, https://github.com/JoinColony/purser/issues/184

This has a fix with an open PR: https://github.com/LedgerHQ/ledgerjs/pull/241